src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1389,6 +1394,47 @@
             }
         },
 
+        "reportingDescriptorTranslation": {
+            "description": "tbd",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "id": {
+                    "description": "TBD matches reportingDescriptor.id",
+                    "type": "string"
+                },
+
+                "guid": {
+                    "description": "A unique identifer for the reporting descriptor in the form of a GUID, matches reportingDescriptor.guid",
+                    "type": "string"
+                },
+
+                "shortDescription": {
+                    "description": "A concise description of the report. Should be a single sentence that is understandable when visible space is limited to a single line of text.",
+                    "$ref": "#/definitions/multiformatMessageString"
+                },
+
+                "fullDescription": {
+                    "description": "A description of the report. Should, as far as possible, provide details sufficient to enable resolution of any problem indicated by the result.",
+                    "$ref": "#/definitions/multiformatMessageString"
+                },
+
+                "messageStrings": {
+                    "description": "A set of name/value pairs with arbitrary names. Each value is a multiformatMessageString object, which holds message strings in plain text and (optionally) Markdown format. The property namesshould be a subset of the names in reportingDescriptor.messageStrings.",
```

Comment:
>description [](start = 21, length = 11)

Replace the last sentence with: "The property names are a subset of the property names in the messageStrings property of the reportingDescriptor object to which this translation belongs